data-prepper-plugins/mongodb/src/main/java/org/opensearch/dataprepper/plugins/mongo/stream/StreamWorker.java

```diff
@@ -227,6 +248,19 @@ public void processStream(final StreamPartition streamPartition) {
         }
     }
 
+    private boolean isCRUDOperation(final OperationType operationType) {
+        return OperationType.INSERT == operationType ||
```

Comment:
MNT: suggest maintain a set of CRUD operation and check whether it is in the set

Reply:
Done